mtags-shared/src/main/scala/scala/meta/internal/pc/CompilerAccess.scala

```diff
@@ -28,8 +28,18 @@ abstract class CompilerAccess[Reporter, Compiler](
     config: PresentationCompilerConfig,
     sh: Option[ScheduledExecutorService],
     newCompiler: () => CompilerWrapper[Reporter, Compiler],
-    shouldResetJobQueue: Boolean
+    shouldResetJobQueue: Boolean,
+    additionalReportingData: () => String
 )(implicit ec: ExecutionContextExecutor, rc: ReportContext) {
+
+  def this(
```

Comment:
Is there a difference between this and giving a default value to the `additionalReportingData` arg?

Reply:
It seems there is for the JVM, it fails in with the default argument.